app/marketing/views.py

```diff
@@ -153,6 +155,15 @@ def privacy_settings(request):
     return TemplateResponse(request, 'settings/privacy.html', context)
 
 
+def record_form_submission(request, obj, _type):
```

Comment:
Can we rename `_type` to `record_type` or `submission_type` to avoid confusion as a private var?

Reply:
updated